lib/manageArtifacts.js

```diff
@@ -67,10 +67,10 @@ const defineVariableOperation = (variable) => {
         }
     }
     catch (error) {
-        throw Error('Error type');
+        throw Error(`Type error: ${error}`);
```

Comment:
What are the likely errors there?
Need more comments above to explain the different scenarios

Reply:
Just added a comment about it.